Tests/test_builtinfunc.py

```diff
@@ -21,718 +24,716 @@
 
 items = globals().items() #4716
 
-import sys
-def cp946():
-    builtins = __builtins__
-    if type(builtins) is type(sys):
-        builtins = builtins.__dict__
-    
-    if "hasattr" not in builtins:
-        raise "hasattr should be in __builtins__"
-    if "HasAttr" in builtins:
-        raise "HasAttr should not be in __builtins__"
+class BuiltinsTest1(unittest.TestCase):
+    def cp946(self):
+        builtins = __builtins__
+        if type(builtins) is type(sys):
+            builtins = builtins.__dict__
+        
+        self.assertIn('hasattr', builtins, 'hasattr should be in __builtins__')
+        self.assertNotIn('HasAttr', builtins, 'HasAttr should not be in __builtins__')
 
-cp946()
-if sys.platform=="cli":
-    import clr
-cp946()
+    def test_no_clr_import(self):
+        self.cp946()
```

Comment:
I'll rename the methods, there is this note in the unittest docs:

"Note that the order in which the various test cases will be run is determined by sorting the test function names with respect to the built-in ordering for strings."